test-requirements.txt

```diff
@@ -3,3 +3,5 @@ pbr
 tox
 bandit
 nose
+coverage
+nose-timer
```

Comment:
I think it's better to make this in another patch.

Reply:
see #20